stock_outgoing_shipment_report/models/stock_outgoing_shipment_report.py

```diff
@@ -20,6 +20,7 @@
     "customer_delivery_note": ["Char", 9],
     "client_order_ref": ["Char", 30],
     "memo": ["Char", 9],
+    "date_created": ["Date", 20],
```

Comment:
```suggestion
```
Let's remove this as it's not expected to be exported.

Reply:
I understand.
Deleted it. Thank you.